package.json

```diff
@@ -169,6 +169,7 @@
     "cozy-device-helper": "^1.10.0",
     "cozy-doctypes": "^1.69.0",
     "cozy-harvest-lib": "^6.7.3",
+    "cozy-intent": "^1.1.2",
```

Comment:
"~1.0" or "1.x" ?

Reply:
1.1.2 has a bugfix, it should not be possible to allow an earlier version I think

Reply:
>= 1.1.2 then ?

Reply:
alright for me